locations/spiders/our_airports.py

```diff
@@ -26,7 +26,7 @@ def parse_row(self, response: Response, row: dict):
         item["name"] = row["name"]
         item["lat"] = row["latitude_deg"]
         item["lon"] = row["longitude_deg"]
-        item["extras"]["ele"] = row["elevation_ft"]
+        item["extras"]["ele:ft"] = row["elevation_ft"]
```

Comment:
Isn't the accepted custom to append "ft" to the value?

```suggestion
        item["extras"]["ele"] = f"{row["elevation_ft"]} ft"
```

Reply:
Usually `'`, https://wiki.openstreetmap.org/wiki/Map_features/Units but ele is explicitly excluded, it has to be meters.  People have tried to remove the exception multiple times https://wiki.openstreetmap.org/wiki/Proposal:Ele_with_units

Reply:
Huh, yet another thing that has changed in OSM over the years 😄